jac-uk/admin
# #1500 Improvement on QT history log

## What's included?
Closes #1500 
Changes the way we present the history log to display date and time of each event instead of duration since last event.
This is particularly useful where the test has been taken over multiple days (e.g. main test day plus mop-up day)

## Who should test?
✅ Product owner

## How to test?
Open the QT response page for a completed test. View the History tab and check that the log at the end displays dates & times.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Should display date & time next to each event.
![image.png](https://images.zenhubusercontent.com/5e412648a39708a606c962c9/399cff19-4265-4758-9758-4c02401173e1)

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
